src/utils/std.ml

```diff
@@ -751,6 +751,13 @@ module Shell = struct
     List.rev !comps
 end
 
+module System_command = struct
+  let unix = ref @@
```

Comment:
With the other comments in mind, I think a better name would be:

`Std.System.run_in_directory` or something similar.

Reply:
You've requested Windows hook as well. Should we name it `run_in_directory_unix` and `run_in_directory_windows`?